nilearn/decomposition/tests/test_canica.py

```diff
@@ -236,7 +236,6 @@ def test_masker_attributes_with_fit(canica_data, mask_img):
     canica = CanICA(n_components=3, mask=mask_img, random_state=0)
     canica.fit(canica_data)
 
-    assert canica.mask_img_ == mask_img
```

Comment:
removing as mask_img_ may now be different from mask_img (it may be a binarized version of it)